dahu/core/app/scripts/views/workspace/objects/tooltip.js

```diff
@@ -42,7 +47,8 @@ define([
         },
 
         modelEvents: {
-            'change': 'render'
+            'change': 'render',
+            'change:color': 'changeBackgroundColor'
```

Comment:
event name is not formatted as described in https://github.com/dahuapp/dahu/issues/110